### PR TITLE
Feature/tlt updates

### LINKED ISSUE #7: Expanding message size when no emojis
All messages are blown up to be a bit higher than they should be, due to the non-breaking-space injected at the end of every message. only add that space in when needed, not always, to prevent this

### PR DESCRIPTION
**Pull request by jkdufair**
Closes #7 #6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,18 +2,34 @@
...
-    "jest": "^24.8.0"
+    "@types/jquery": "^3.5.6",
+    "cross-env": "^7.0.3",
+    "jest": "^24.9.0",
+    "nodemon": "^1.19.1",
+    "tslint": "^5.18.0"
   },
   "dependencies": {
-    "asar": "^2.0.1"
+    "@azure/storage-blob": "^12.0.0",
```
Comment (dsmiller95) [inline package.json:23]: why reference azure storage, is this a modification which stores emojis into azure storage directly?

File: src/payload.ts [hunk 2]
```diff
@@ -0,0 +1,394 @@
+interface StorageData {
+	apiUrl: string
+}
+
+//Make sure the payload is injected only once per context at most
+// @ts-ignore
+if (window.SECRET_EMOJI_KEY != "set") {
+	// @ts-ignore
```
Comment (dsmiller95) [inline src/payload.ts:8]: love the typescript conversion. its a shame the inconsistent spacing makes the diff weird

**Review by dsmiller95 — COMMENTED** [review]
does this work the same as the version on master, for installation locally and as a chrome extension? the changes look promising but are hard to compare in the PR view

**Comment by jkdufair** [comment]
Hey really sorry. Forked the project for our own team's use and had intended to do a PR within the same repo.  Glad to submit anything that looks useful back upstream once it's functional.

**Comment by dsmiller95** [comment]
awesome, I'm happy you and your team are finding use out of this. even better if some of the changes can come back to main

**Pull request closed without merging**